client/.storybook/main.js

```diff
@@ -5,12 +5,14 @@ const toPath = filePath => path.join(process.cwd(), filePath);
 module.exports = {
   staticDirs: ['../packages/host/public'],
   typescript: { reactDocgen: 'react-docgen' },
-  reactOptions: {
-    fastRefresh: true,
-  },
   framework: '@storybook/react',
   core: {
-    builder: '@storybook/builder-webpack5',
+    builder: {
+      name: 'webpack5',
+      options: {
```

Comment:
not part of the fix, and an experimental feature of webpack. but neat - speeds up compilation time in develop mode, at the expense of slightly longer load times, most noticeable if you're switching between different stories